spark-jobs/pom.xml

```diff
@@ -71,6 +72,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- Commons - for Version utilities -->
```

Comment:
Remove this comment. I am going to use commons as well (`TempFIle`) this comment would become misleading.

Reply:
Sure, done.